.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '2.6'
+          ruby-version: '3.1'
```

Comment:
`rubocop` should be tested with the oldest supported Ruby.

Reply:
Ok reverted this 👍